rust/operator-binary/src/superset_controller.rs

```diff
@@ -532,6 +548,13 @@ fn build_rolegroup_config_map(
         rolegroup: rolegroup.clone(),
     })?;
 
+    // By removing the key from `config_properties`, we avoid pasting the Python code into a Python variable as well
+    // (which would be bad)
+    // TODO: Use public constants in operator-rs to reference `EXPERIMENTAL_FILE_HEADER` and `EXPERIMENTAL_FILE_FOOTER`
```

Comment:
```suggestion
```

Reply:
I've removed a part of the comments.

I would leave the `TODO:` until we can use the consts from `operator-rs`

Reply:
I think the comment is valuable and would prefer to keep it ;)
I just meant the duplicated comment below, where we actually don't `remove` anything ;)

Can you please re-add the first comment again?

Reply:
Damn it 😂

First thing tomorrow morning!

Reply:
Done with https://github.com/stackabletech/superset-operator/pull/530/commits/057039d9830443204dc422325d4a6d20e3387fc3